lib/src/pagination/pagination.component.html

```diff
@@ -0,0 +1,99 @@
+                    <button class="page-link" title="previousText">
+                        <span class="nav-action">
+                            <ng-container *ngIf="useTextNav; else angleLeftIcon">
+                                {{ previousText ? previousText : "Previous" }}
+                            </ng-container>
+                            <ng-template #angleLeftIcon>
+                                <svg name="angle-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 192 512">
+                                    <path
+                                        d="M25.1 247.5l117.8-116c4.7-4.7 12.3-4.7 17 0l7.1 7.1c4.7 4.7 4.7 12.3 0 17L64.7 256l102.2 100.4c4.7 4.7 4.7 12.3 0 17l-7.1 7.1c-4.7 4.7-12.3 4.7-17 0L25 264.5c-4.6-4.7-4.6-12.3.1-17z"
+                                    />
+                                </svg>
+                            </ng-template>
+                        </span>
+                        <span class="sr-only">{{ previousText || "Previous" }}</span>
+                    </button>
+                </li>
+            </ng-container>
+            <ng-container *ngFor="let num of list; let i = $index">
+                <li class="page-item" [ngClass]="{ active: value === num }" (click)="handleOnChange(num)" value="num">
+                    <button class="page-link" [ngClass]="{ active: value === num }">
+                        <span class="nav-num">{{ num }}</span>
+                        <span class="sr-only">{{ num }}</span>
+                    </button>
+                </li>
+            </ng-container>
+            <ng-container *ngIf="value !== pagingSize">
+                <li class="page-item" (click)="handleOnChange(value + 1)">
+                    <button class="page-link" title="nextText">
+                        <span class="nav-action">
+                            <ng-container *ngIf="useTextNav; else angleRightIcon"> {{ nextText ? nextText : "Next" }} </ng-container>
+                            <ng-template #angleRightIcon>
+                                <svg name="angle-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 192 512">
+                                    <path
+                                        d="M166.9 264.5l-117.8 116c-4.7 4.7-12.3 4.7-17 0l-7.1-7.1c-4.7-4.7-4.7-12.3 0-17L127.3 256 25.1 155.6c-4.7-4.7-4.7-12.3 0-17l7.1-7.1c4.7-4.7 12.3-4.7 17 0l117.8 116c4.6 4.7 4.6 12.3-.1 17z"
+                                    />
+                                </svg>
+                            </ng-template>
+                        </span>
+
+                        <span class="sr-only">{{ nextText || "Next" }}</span>
+                    </button>
+                </li>
+            </ng-container>
+            <ng-container *ngIf="value !== pagingSize && useFirstAndLast">
+                <li class="page-item" (click)="handleOnChange(pagingSize)">
+                    <button class="page-link" title="lastText">
```

Comment:
Don't forget the binding
```suggestion
                    <button class="page-link" [title]="lastText">
```